runtime-complexity/memoization.md

```diff
@@ -123,28 +122,27 @@ Ans: create a hash table to store the data to be used again when called upon.
 ```if (memo[num]) return memo[num]```
 
 4. Write the rest of the function checking for 0 and then calling the memo if it exists. 
-  ```
-  if (num <= 1) return 1;
+```
```

Comment:
Get back to previous indentation. Same for line no: 126, 128, 134 and 136 to 138.